torch/distributed/optim/functional_sgd.py

```diff
@@ -120,7 +123,9 @@ def step(self, gradients: List[Optional[Tensor]]):
                   momentum=momentum,
                   lr=lr,
                   dampening=dampening,
-                  nesterov=self.nesterov)
+                  nesterov=self.nesterov,
+                  maximize=self.maximize,
+                  )
```

Comment:
indent of this line should align with `F.sgd`?

Reply:
I tried that but it kept failing the flake8 tests :(
I haven't figured out how to get ALE linting to work in vim, so I wasn't able to run an autoformatter to resolve the issues

Reply:
Since the change would break flake8, I'll land as-is.

Reply:
I have force pushed a reset to the commit that both of you approved.